docs/creating_integration_manifest.md

```diff
@@ -179,6 +179,34 @@ MQTT discovery works by subscribing to MQTT topics specified in the manifest.jso
 }
 ```
 
+## DHCP
+
+If your integration supports discovery via dhcp, you can add the type to your manifest. If the user has the `dhcp` integration loaded, it will load the `dhcp` step of your integration's config flow when it is discovered. We support DHCP discovery by the `hostname` and [OUI](https://en.wikipedia.org/wiki/Organizationally_unique_identifier). The manifest value is a list of matcher dictionaries, your integration is discovered if all items of any of the specified matchers are found in the DHCP data. It's up to your config flow to filter out duplicates.
+
+If the integration supports `zeroconf` or `ssdp`, these should be preferred over `dhcp` as it generally offers a better
+user experience.
+
+The following example has one matcher consisting of three items, all of which must match for discovery to happen by this config.
```

Comment:
In this example case, it's quite confusing as it's impossible to meet the requirement of matching _all_ these mac addresses for a single device, is this intentional?

Reply:
I've updated the text to make it clear its passively listening.